concrete-core/src/backends/core/private/crypto/lwe/ciphertext_seeded.rs

```diff
@@ -0,0 +1,153 @@
+#[cfg(feature = "serde_serialize")]
+use serde::{Deserialize, Serialize};
+
+use concrete_commons::numeric::Numeric;
+use concrete_commons::parameters::{LweDimension, LweSize};
+
+use crate::backends::core::private::crypto::secret::generators::EncryptionRandomGenerator;
+use crate::backends::core::private::math::random::{RandomGenerable, RandomGenerator, Uniform};
+use crate::backends::core::private::math::tensor::AsMutTensor;
+
+use super::{LweBody, LweCiphertext};
+
+/// A seeded ciphertext encrypted using the LWE scheme.
+#[cfg_attr(feature = "serde_serialize", derive(Serialize, Deserialize))]
+#[derive(Debug, Clone, PartialEq)]
+pub struct LweSeededCiphertext<Scalar> {
+    pub(super) body: LweBody<Scalar>,
+    pub(super) lwe_dimension: LweDimension,
+    pub(super) seed: u128,
+    pub(super) shift: usize,
+}
```

Comment:
Having a single `CryptoSeed` seems like a good idea! But I don't get why it couldn't be used exactly as it is right now, since it would only be a wrapper around what already exists?